packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -189,6 +190,28 @@ class Scrollable extends StatefulWidget {
   ///    exclusion.
   final bool excludeFromSemantics;
 
+  /// {@template flutter.widgets.scrollable.primaryScrollShortcut}
+  /// Indicates whether this [Scrollable] should be used as the
+  /// [PrimaryScrollShortcut].
+  ///
+  /// When invoking a [ScrollAction] via a keyboard key combination that maps to
+  /// a [ScrollIntent], the [primaryFocus] is used to find the current
+  /// Scrollable that encloses it. If a Scrollable cannot be found, the
+  /// PrimaryScrollShortcut serves as a fallback, allowing for a default
+  /// scrolling shortcut. Only one Scrollable can be registered with the
+  /// PrimaryScrollShortcut at a time.
+  ///
+  /// When true, this Scrollable will be provided to the PrimaryScrollShortcut
+  /// for ScrollActions. Defaults to false.
+  ///
+  /// See also:
+  ///
+  ///  * [PrimaryScrollShortcut], an inherited widget that provides the ability
+  ///    for default ScrollActions to take place outside of th context fo the
+  ///    current focus.
+  /// {@endtemplate}
+  final bool primaryScrollShortcut;
```

Comment:
It think this needs a better name, I found it hard to know what it was before I read this documentation block.

Some ideas:
 - `isDefaultPrimaryScrollable`
 - `isFallbackScrollable`
 - `isPrimaryScrollable`
 - `isDefaultScrollable`
 - Or something else...these don't feel perfect either.

I feel like it doesn't need the "Shortcut" as part of the name, since you could invoke the `ScrollAction` without using a shortcut if you wanted to. It should also probably begin with "is", since it's a bool.

Reply:
I see what you mean, I'll experiment with some new names. Thanks for the feedback! :)